### PR TITLE
Created a new Phase2 Scenario (2026D55 ) with an updated Phase2 Muon Geometry (M5) 

### PR DESCRIPTION
**Pull request by slomeo**
#### PR description:

Scenario 2026D55 = Scenario 2026D54 but  with "M5" instead of "M4"
In "M5" there are:
1) YE3 thickness: fixed (PR #29266 ) now also for 2026
2) "Hidden Overlaps" in the Muon System: fixed (PR #29379 ) now also for 2026
3) New version of iRPC (by @bapavlov) 

Fixed all overlaps due to iRPC (check by cmsShow)
Test of runTheMatrix ok

### CODE REVIEW

File: Configuration/Geometry/README.md [hunk 1]
```diff
@@ -54,5 +55,6 @@ Several detector combinations have been generated:
 * D52 = T18+C9+M4+I10+O4+F2
 * D53 = T15+C9+M4+I12+O4+F2
 * D54 = T19+C9+M4+I10+O4+F2
+* D54 = T19+C9+M5+I10+O4+F2
```
Comment (kpedro88) [inline Configuration/Geometry/README.md:58]: typo D54->D55

**Comment by kpedro88** [comment]
If the workflow is expected to run to completion, can you add the ttbar workflow number here: https://github.com/cms-sw/cmssw/blob/master/Configuration/PyReleaseValidation/python/relval_2026.py
then it can be tested in the PR

File: Configuration/Geometry/README.md (hunk 1 above)
Comment (slomeo) [reply Configuration/Geometry/README.md]: Done

**Comment by slomeo** [comment]
it is already present (i.e numWFIB.extend([26234.0]) #2026D55 )

**Comment by kpedro88** [comment]
test parameters
workflows = 26234.0

**Comment by kpedro88** [comment]
please test

**Comment by cvuosalo** [comment]
`SimG4Core/PrintGeomInfo/test/python/g4OverlapCheck_cfg.py` should be removed from this PR. It is not actually changed.

**Comment by slomeo** [comment]
Done and made squash

**Comment by chayanit** [comment]
please test

**Comment by kpedro88** [comment]
+upgrade

**Comment by cvuosalo** [comment]
Why is `SimG4Core/PrintGeomInfo/test/python/g4OverlapCheck_cfg.py` being deleted? Is there something wrong with it?

**Comment by slomeo** [comment]
I perfomed a git-rm, is it a problem? My idea was to remove it from branch..

**Comment by cvuosalo** [comment]
@slomeo This PR is currently proposing to delete `SimG4Core/PrintGeomInfo/test/python/g4OverlapCheck_cfg.py` from CMSSW. That is what `git-rm` means. If you want to leave the file unchanged, you will need to `git add` the original version back to your branch and then `git push`.

**Comment by slomeo** [comment]
@cvuosalo : done

**Comment by civanch** [comment]
please test

**Comment by slomeo** [comment]
@cvuosalo  @ianna  @bsunanda @civanch : what's wrong? I added only g4OverlapsCheck.. and yesterday all tests were fine.
> -1
> 
> Tested at: [ff4addd](https://github.com/cms-sw/cmssw/commit/ff4addd22e1dbaef9beba581593c1a2b9f8fb33d)
> 
> CMSSW: CMSSW_11_1_X_2020-04-17-2300
> SCRAM_ARCH: slc7_amd64_gcc820
> You can see the results of the tests here:
> https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-41c9f3/5761/summary.html
> 
> I found follow errors while testing this PR
> 
> **Failed tests: UnitTests**
> 
>     * **Unit Tests**:
> 
> 
> I found errors in the following unit tests:
> 
> ---> test test2026Geometry had ERRORS

**Comment by civanch** [comment]
@slomeo , test2026Geometry test has error for cmsExtendedGeometry2026D55XML_cfi.py - the result of test is different from previous. Because you modify 2026D55, it is not a surprise.

Now question to @cvuosalo should the test be modified in this PR or we ignore this failer?

**Comment by kpedro88** [comment]
@civanch there is never any circumstance in which errors in test2026Geometry can be ignored. This test is crucial to ensure consistency in the phase 2 geometries.

@slomeo any changes to the auto-generated geometry configs should be made by editing dict2026Geometry.py and then rerunning generate2026Geometry.py.

also, please squash your commits in this PR to remove the deletion and recreation of g4OverlapCheck_cfg.py from the history.

**Comment by kpedro88** [comment]
@slomeo @civanch actually, the unit test failed because #29482, which also made changes to dict2026Geometry.py, was merged in the meantime. Please rebase this PR on the latest IB and then rerun generate2026Geometry.py. (This demonstrates why the unit test is essential.)

**Comment by slomeo** [comment]
@kpedro88  made squash and run generate2026Geometry.py in new IB

**Comment by civanch** [comment]
please test

**Comment by kpedro88** [comment]
+upgrade

**Comment by boudoul** [comment]
Hello @kpedro88 , @slomeo , I have a question : why for each new version of the Muon were are creating yet another scenario instead of updating the existing one ?  Thanks

**Comment by kpedro88** [comment]
@boudoul the policy for phase 2 geometries is that once a geometry is used in any official production (TDR etc.), it is frozen and can't be changed. M4 is included in D49 (the HLT TDR baseline). Therefore, any updates must go into a new geometry version. This is the only way we have found to maintain sanity and consistency. Groups often want to go back and study the differences between new and old geometry versions, on the basis of TDR performance plots.

**Comment by boudoul** [comment]
thanks @kpedro88 for the super quick answer! 
I understand that you do  not want to touch neither M4 nor D49 .

but why not updating just D54 with the new M5 , instead of creating the new scenario 2026D55 (= Scenario 2026D54 but with "M5" instead of "M4")

**Comment by kpedro88** [comment]
We also try to vary "one at a time" in development geometries. Also varying the muon system in D54 might disrupt the ongoing work by the tracker team for T19, etc.

**Comment by boudoul** [comment]
ok thanks . 
Maybe we could also ask @slomeo  to update the title of this PR to make it a bit more explicit , something like "creating a new phase2 scenario (2026D55 ) with an updated phase2 Muon geometry  (M5) " ?

**Comment by cvuosalo** [comment]
+1

**Comment by slomeo** [comment]
@boudoul : title updated

**Comment by silviodonato** [comment]
+operations

**Comment by boudoul** [comment]
> @boudoul : title updated

Thanks!

**Comment by chayanit** [comment]
+1

**Comment by silviodonato** [comment]
+1

**Pull request merged**